src/components/common/FilterButtons.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import { useDispatch } from 'react-redux';
+
+export default function FilterButtons({
+    cols = 2,
+    filterName = '',
+    selectedValues = [],
+    updateFunc = () => null,
+    values = [],
+} = {}) {
+    const dispatch = useDispatch();
+
+    const toggleValue = value => {
+        const isSelected = selectedValues.includes(value);
+
+        if (isSelected)
+            dispatch(
+                updateFunc({
+                    field: filterName,
+                    value: selectedValues.filter(val => val !== value),
```

Comment:
lodash has a nice util function for this btw https://lodash.com/docs/4.17.15#without doesn't make much difference here but just to mention it